test/unit/email_regex_test.rb

```diff
@@ -0,0 +1,127 @@
+# encoding: utf-8
+# rubocop:disable all
```

Comment:
Don't disable Rubocop in new files.

Reply:
Sure. i am beginning to like rubocop. Never used it but it is really nice to have the code to be in order.